### PR TITLE
Dums 63 timer for votes

### PR DESCRIPTION
**Pull request by alexunique0519**
Added timer in roomInfo component. When the host start the timer, all joined clients will get the countdown timer showing on the screen.  If a user joins after the timer starts, he/she will get synced timer info on the screen.

### CODE REVIEW

File: components/CountdownTimer.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import Countdown from 'react-countdown'
```
Comment (DaceyTom2) [inline components/CountdownTimer.js:2]: Add a test file for this component with basic tests

File: package.json [hunk 2]
```diff
@@ -31,6 +31,7 @@
     "prom-client": "^12.0.0",
     "prop-types": "^15.8.1",
     "react": "17.0.2",
+    "react-countdown": "^2.3.2",
```
Comment (DaceyTom2) [inline package.json:34]: This package isn't well maintained or popular https://github.com/ndresx/react-countdown
It is okay for a prototype but for a real product we should look into better packages

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]
Add some tests and should be good

File: components/CountdownTimer.js (hunk 1 above)
Comment (alexunique0519) [reply components/CountdownTimer.js:2]: test added

**Review by DaceyTom2 — APPROVED** [review]

**Pull request merged**